database/migrations/2024_06_18_073939_add_high_score_to_quiz_attempts_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('quiz_attempts', function (Blueprint $table) {
+            $table->integer('high_score')->default(0);
```

Comment:
you don't need a new field in the quiz attempt you can get the highest score from the making a query in the quiz attempts and storing the highest score in a local variable than comparing with the actual quiz score and if the score from the actual quiz is bigger than any of the scores of that users quiz attempt you only display a message by saying "You have a new high score" . "$score"